changelog.md

```diff
@@ -1,3 +1,116 @@
+* image-elements: remove mimeType from artifact ([#13265](https://github.com/GoogleChrome/lighthouse/pull/13265))
+* image-size-responsive: remove elidedUrl, elide url property instead ([#13226](https://github.com/GoogleChrome/lighthouse/pull/13226))
+
+## Report
+
+* introduce the new report api, add dom.rootEl ([#13277](https://github.com/GoogleChrome/lighthouse/pull/13277), [#13279](https://github.com/GoogleChrome/lighthouse/pull/13279), [#13361](https://github.com/GoogleChrome/lighthouse/pull/13361))
+* make denser. changes to typography, group descriptions, more ([#13249](https://github.com/GoogleChrome/lighthouse/pull/13249))
+* display final screenshot prominently ([#13123](https://github.com/GoogleChrome/lighthouse/pull/13123))
+* redesign runtime settings ([#13125](https://github.com/GoogleChrome/lighthouse/pull/13125), [#13350](https://github.com/GoogleChrome/lighthouse/pull/13350))
+* hidden group backcompat ([#13310](https://github.com/GoogleChrome/lighthouse/pull/13310))
+* help-dialog: remove timespan SEO ([#13354](https://github.com/GoogleChrome/lighthouse/pull/13354))
+* order metrics by row ([#13328](https://github.com/GoogleChrome/lighthouse/pull/13328))
+* translate newly added report strings ([#13308](https://github.com/GoogleChrome/lighthouse/pull/13308))
+* group perf audits by details type ([#13241](https://github.com/GoogleChrome/lighthouse/pull/13241))
+* sort audits by weight ([#13053](https://github.com/GoogleChrome/lighthouse/pull/13053))
+* pwa: remove gauge alias ([#13269](https://github.com/GoogleChrome/lighthouse/pull/13269))
+* flow: remove smooth scrolling ([#13317](https://github.com/GoogleChrome/lighthouse/pull/13317))
+* flow: network throttling settings ([#13305](https://github.com/GoogleChrome/lighthouse/pull/13305))
+* flow: use filmstrip thumbnail in navigation ([#13283](https://github.com/GoogleChrome/lighthouse/pull/13283))
+* flow: fix report anchors ([#13233](https://github.com/GoogleChrome/lighthouse/pull/13233))
+* flow: full page screenshot renderer ([#13276](https://github.com/GoogleChrome/lighthouse/pull/13276))
+* flow: category tooltip highest impact ([#13230](https://github.com/GoogleChrome/lighthouse/pull/13230))
+* flow: import lhr strings ([#13215](https://github.com/GoogleChrome/lighthouse/pull/13215))
+* flow: i18n formatter ([#13190](https://github.com/GoogleChrome/lighthouse/pull/13190))
+* flow: mock styles ([#13220](https://github.com/GoogleChrome/lighthouse/pull/13220))
+
+## ⛏️👷 Fraggle Rock
+
+  Support for auditing user flows ([#11313](https://github.com/GoogleChrome/lighthouse/issues/11313))
+
+* fr: handle 0 throughput in timespan ([#13323](https://github.com/GoogleChrome/lighthouse/pull/13323))
+* fr: clone default categories to avoid modification ([#13337](https://github.com/GoogleChrome/lighthouse/pull/13337))
+* fr: add timespan support to h2 ([#12814](https://github.com/GoogleChrome/lighthouse/pull/12814))
+
+## Deps
```

Comment:
Emojis for all of these like on https://github.com/GoogleChrome/lighthouse/releases/tag/v8.0.0?